README.md

```diff
@@ -138,6 +138,6 @@ I would like to thank...
 
 ## 📝 License <a name="license"></a>
 
-This project is [MIT](./LICENSE) licensed.
+This project is [MIT](https://github.com/chepkok3/Leaderboard/blob/final-touches/MIT%20License) licensed.
 
```

Comment:
Note: This is only optional because It wasn't mentioned by the previous reviewer, but I recommend that you follow these points every time you document your README.
 
[OPTIONAL] Here are key points to consider when documenting your README.

- The template logo should be removed. Optionally, the Microverse logo can be replaced with your own project logo if you have one. ✔️
- The only optional section is the FAQ section. If there's no information in the FAQ section then it should be deleted. ✔️ 
***However, since you deleted the FAQ section, it would be best if you also removed it from the table of contents.***
- All main headers from the [README template](https://github.com/microverseinc/readme-template) should be present. Even if there's no information for a section or sub-section, rather leave texts like "NA", "None" or "coming soon" than delete it. ✔️ 
- Future Features section should include 1-3 potential features to be added. ✔️ 
-  Each section description should be removed and replaced with their custom content. The section descriptions are meant to guide you in documenting your README. They are not meant to be part of the final documentation. ✖️ 

For example:
![image](https://user-images.githubusercontent.com/38088818/216551345-4cb3cf19-9d8d-47d5-95be-ac8d47de0049.png)
![image](https://user-images.githubusercontent.com/38088818/216551482-2a274f75-4946-434a-97ea-746434b0a464.png)
![image](https://user-images.githubusercontent.com/38088818/216551578-aae6a149-69c8-4a13-bb22-4639defb421c.png)
***Some sections still have the description, please remove them***

- The License suggestion should be removed. ✔️ 
- There should not be any blank sections that do not have content. ✖️ 

Kindly document the `Acknowledgments` section.
![image](https://user-images.githubusercontent.com/38088818/216764621-55a10d26-d53f-46ce-9dfe-485af2277370.png)


- There should not be any broken links in the README file. ✔️ 
- Delete the live demo section if the application is not deployed. If project requirements say to deploy the project, the section should be restored and the link updated. ✖️ 

![image](https://user-images.githubusercontent.com/38088818/216765466-fcad5ef6-30a6-4872-9fdc-e31a11093558.png)

Reply:
@KDlamini Noted👍🏿